src/extensionVariables.ts

```diff
@@ -23,8 +24,8 @@ namespace extEvents {
 }
 
 export namespace extActions {
-    export let refreshWorkspaceTree: () => void;
-    export let refreshAzureTree: () => void;
+    export let refreshWorkspaceTree: (data?: ResourceGroupsItem | ResourceGroupsItem[] | null | undefined | void) => void;
```

Comment:
Do we need `null | undefined | void`? It seems a little excessive. The parameter is also already optional as it is.

Reply:
I'm keeping consistent with the `onDidChangeTreeData` event data type.

```typescript
onDidChangeTreeData?: Event<T | T[] | undefined | null | void>;
```